internal/app/api/v1/testworkflows.go

```diff
@@ -371,34 +388,37 @@ func (s *TestkubeAPI) ExecuteTestWorkflowHandler() fiber.Handler {
 		var results []testkube.TestWorkflowExecution
 		var errs []error
 
-		request.TestWorkflowExecutionName = strings.Clone(c.Query("testWorkflowExecutionName"))
-		concurrencyLevel := scheduler.DefaultConcurrencyLevel
-		workerpoolService := workerpool.New[testworkflowsv1.TestWorkflow, testkube.TestWorkflowExecutionRequest,
-			testkube.TestWorkflowExecution](concurrencyLevel)
-		requests := []workerpool.Request[testworkflowsv1.TestWorkflow, testkube.TestWorkflowExecutionRequest, testkube.TestWorkflowExecution]{
-			{
-				Object:  *workflow,
-				Options: request,
-				ExecFn:  s.TestWorkflowExecutor.Execute,
-			},
-		}
-
-		go workerpoolService.SendRequests(requests)
-		go workerpoolService.Run(ctx)
-
-		for r := range workerpoolService.GetResponses() {
-			results = append(results, r.Result)
-			if r.Err != nil {
-				errs = append(errs, r.Err)
+		if len(testWorkflows) != 0 {
+			request.TestWorkflowExecutionName = strings.Clone(c.Query("testWorkflowExecutionName"))
+			concurrencyLevel, err := strconv.Atoi(c.Query("concurrency", strconv.Itoa(scheduler.DefaultConcurrencyLevel)))
```

Comment:
Should we support concurrency there? It will be only for scheduling, not actually executing concurrently

Reply:
Ok, we can remove it